### PR TITLE
Release 20210201

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] Correct Markdown and formatting
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

-----
[View rendered CONTRIBUTING.md](https://github.com/publiccodenet/about/blob/release-20210201/CONTRIBUTING.md)
[View rendered activities/index.md](https://github.com/publiccodenet/about/blob/release-20210201/activities/index.md)
[View rendered activities/live-streaming/index.md](https://github.com/publiccodenet/about/blob/release-20210201/activities/live-streaming/index.md)
[View rendered activities/live-streaming/manage-live-stream.md](https://github.com/publiccodenet/about/blob/release-20210201/activities/live-streaming/manage-live-stream.md)
[View rendered activities/live-streaming/post-production.md](https://github.com/publiccodenet/about/blob/release-20210201/activities/live-streaming/post-production.md)
[View rendered activities/live-streaming/prepare-live-stream.md](https://github.com/publiccodenet/about/blob/release-20210201/activities/live-streaming/prepare-live-stream.md)

### CODE REVIEW

File: activities/live-streaming/manage-live-stream.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+explains: how to run a live stream
+---
+
+# Managing a live stream
+
+When running a [live stream](index.md) of 'Let's talk about  public code' there are some steps to remember.
+
+- Test everything before going live.
+- Prepare tabs with easy access to moderation tools of the different platforms to be able to intervene in case of offensive commenting. Preferably, this is done by a staff member not in the actual interview.
+- Meet with the guest in [StreamYard](../tool-management/streamyard.md] 30 minutes before going live.
```
Comment (bvhme) [inline activities/live-streaming/manage-live-stream.md:12]: Markdown syntax:

```suggestion
- Meet with the guest in [StreamYard](../tool-management/streamyard.md) 30 minutes before going live.
```

**Review by bvhme — CHANGES_REQUESTED** [review]
Looks good bar some missed broken link.

**Comment by ElenaFdR** [comment]
This release includes errors I fixed in #741, which were ignored or reintroduced after I fixed them. I didn't approve #741, and this release doesn't reflect our usual style.

**Comment by Ainali** [comment]
> This release includes errors I fixed in #741, which were ignored or reintroduced after I fixed them. I didn't approve #741, and this release doesn't reflect our usual style.

Yes, but to me there is something strange going on, because in [the list of commits](https://github.com/publiccodenet/about/pull/741/commits) I cannot see when they were reintroduced.

**Comment by ElenaFdR** [comment]
@Ainali Looking at #741 commit-by-commit, I was inconsistent in catching all instances of the same problem. I'll fix them here. Sorry for the confusion.

**Comment by ElenaFdR** [comment]
Only 2 issues left, and neither is fatal: 
- livestream bullets use dashes, not asterisks, which is our usual
- some livestream pages have sub-bullets within long bulleted lists. These can be hard to read; it's usually a sign the content presentation needs a rethink.

File: activities/live-streaming/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Get in touch directly with key figures of top projects. Possible way in to steward or make them members of the Foundation for Public Code.
+* Get more social media followers since we are going to use those networks for this project.
+
+## Formats, length and schedule
+
+1. Frequency: Ambition to make it at least monthly.
+2. Format: Two different formats - one for codebases, one for the Foundation. For the codebases it is an interview made by two codebase stewards - ideally the community ones - with someone relevant from a codebase. For the Foundation, the format is a conversation between two persons from the foundation. This can be theoretical, about ideas and principles or about news.
+3. Language: English.
+4. Script: We get several questions ready before each episode. This builds familiarity with our approach and can also reveal similarities and differences between codebases.
+5. Length and schedule: Minimum 20 minute livestream during business hours - starting at 14.00 UTC if possible. Promise that the podcast will be live by close of business the next day. Aim to hold this on Tuesdays.
```
Comment (bvhme) [inline activities/live-streaming/index.md:22]: I don't see a reason for this to be an ordered list. I recommend either making it an unordered list or just removing the list aspect as it is basically separate statements.

File: activities/live-streaming/manage-live-stream.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* When it's time to start the stream, press Go live and quickly start the intro video.
+* Start by only having the interviewers on screen.
+* Remind the audience what the Foundation for Public Code does and how to contact us in an easy way.
+* Present the guest and bring them on screen.
+* Run the interview.
+* Bring up interesting questions from the audience on the screen and read them out loud to the interviewee.
+* Remember that there will be a podcast version afterwards, so make sure to explain things so that someone only listening to the show will understand (in case of shared screens or funny things happening on camera).
+* The last question we always ask our guest is "Please name someone you think we should interview in our upcoming episodes, and explain why?"
+* Remind the audience at the end of ways of keeping in touch with us.
+* When the show is over start the outro video and quickly press Stop recording.
```
Comment (bvhme) [inline activities/live-streaming/manage-live-stream.md:23]: This on the other hand is a list of steps that I'm guessing should be taken in order and thus might be better served being ordered.

**Review by bvhme — DISMISSED** [review]
Great additions

File: activities/live-streaming/index.md (hunk 2 above)
Comment (ElenaFdR) [reply activities/live-streaming/index.md:22]: cc @Ainali @AlbaRoza

File: activities/live-streaming/manage-live-stream.md (hunk 3 above)
Comment (ElenaFdR) [reply activities/live-streaming/manage-live-stream.md:23]: cc @Ainali @AlbaRoza

**Review by bvhme — APPROVED** [review]
Great! Good to go I think.

**Pull request merged**